### ISSUE TITLE
Malware warning from Windows defender when attempting to upgrade to 3.9.3.0

### ISSUE THREAD

**Issue by odog8** [TeamShinkansen/Hakchi2-CE#421]
**Describe the bug**  

Windows defender detects malware in latest build of hakchi
**To Reproduce**  

upgrade to 3.9.3
**Expected behavior**  

no malware warning
**Screenshots**  

**Additional context**

**Comment by DanTheMan827** [comment]
Safe screen or actual malware detection?

**Comment by odog8** [comment]
actual malware detection

**Comment by odog8** [comment]
Probably a false positive but idk

**Comment by DanTheMan827** [comment]
Weird, I just uploaded the installer and portable version to virus total and defender detected nothing

some other scanners detected Linux “malware”, but it probably confused hakchi with it

**Comment by turbocomppro** [comment]
Yup. Can't update from hakchi or download unless I turn off all the virus scanners.

**Comment by eugeneniemand** [comment]
I get this warning on Win 11 
![image](https://user-images.githubusercontent.com/6813309/162419300-ef0b21c8-e9e8-4188-977b-46615d641ab2.png)

**Comment by DanTheMan827** [comment]
Unfortunately, these kind of things just happen...

It's a case of antivirus _thinking_ there's a virus based on who knows what factors... might by download count, might be that it isn't codesigned, and maybe it's just because it's coming from github

I've seen one antivirus detect it because some of the linux binary files used for the mod itself are UPX packed to save space

**Comment by russellweed** [comment]
I received the same warning on Windows 10 (fully updated), from either downloading the portable release off GitHub or using the built-in auto-updater.

![image](https://user-images.githubusercontent.com/1160446/174223251-7bb59d45-5359-4c1f-975d-80f38f13b801.png)

Maybe just to be on the safe side you guys should make sure you haven't been compromised, either via dependencies or on your build machines?

**Comment by DanTheMan827** [comment]
Which file inside of the archive does it detect?

**Comment by impeeza** [comment]
Unwanted program IS NOT MALWARE (VIRUS, TROJAN, RAMSOMWARE, etc.) is the way in which Microsoft tell you "you are downloading a program what I THINK you shouldn't use" in other words:  The developer has not pay me lots of money for me do not scare the users and give you bad reputation.

So, you have two options:  use a real antivirus, or use a net explorer to scan the file.
do not trust the people here and do not use the software.

**Comment by DanTheMan827** [comment]
@impeeza Here's the thing, I can download the same file and none of my systems detect it...
![image](https://user-images.githubusercontent.com/790119/202559738-cf4e90b9-ef3e-4341-bc26-5d9a52cab83b.png)
It's _probably_ caused by the updater triggering some behavior the antivirus doesn't like.
